doc/source/serve/deployment-graph/http_endpoint_for_dag_graph.md

```diff
@@ -0,0 +1,29 @@
+# Pattern: Http endpoint for dag graph
+
+The example shows how to add endpoint for dag graph
```

Comment:
How about something like 

"This example shows how to configure ingress component of the deployment graph, such as HTTP endpoint prefix, HTTP to python object input adapter and number of ingress replicas"